include/utils/StkHelpers.h

```diff
@@ -102,14 +102,14 @@ get_node_field(
 void register_scalar_nodal_field_on_part(
```

Comment:
This looks redundant for what the `FieldManager` does. These functions look like candidates for the chopping block in the future.

Reply:
Yes, I saw that register_scalar_nodal_field_on_part was only used in a couple of places but was not sure if it was the future or the past. I'll remove it on the next merge request.

Yes, This is just the first of a few merge requests to replace Part* with PartVector. Just wanted to get this in and work on other things for a few days.